### PR TITLE
msys2-runtime: leave cygwin- pipe names as is

### PR DESCRIPTION
**Pull request by forgottenswitch**
This is a patch for MSYS runtime.
Allows for simpler is-it-mintty-stdin pipe name test.

Implied on mingw-w64 mailing list.
Untested.

### CODE REVIEW

**Comment by forgottenswitch** [comment]
makepkg applies the patch, but build fails

File: msys2-runtime/PKGBUILD [hunk 1]
```diff
@@ -49,7 +49,7 @@ source=('msys2-runtime'::git://sourceware.org/git/newlib-cygwin.git#tag=cygwin-$
         0024-machine-_types.h-__blkcnt_t-should-be-signed.patch)
 sha256sums=('SKIP'
             '7b24f901bddfb4b92ed2982d8608e8a9fdd1d2e523ae028115b5f5e8d710fea4'
-            '2683c45d3d964c55738dc65c9897fe6b8344ba20ead4102c432c46e382923625'
+            'SKIP'
```
Comment (mati865) [inline msys2-runtime/PKGBUILD]: There should be checksum

**Comment by mati865** [comment]
Also increase `pkgrel`.

**Comment by mati865** [comment]
It's based on outdated revision.
You have to rebase it on https://github.com/Alexpux/MSYS2-packages/commit/79eddc9fdfe42fd1871e3d0810d43b47b95a3c40 or it will not merge.

**Pull request closed without merging**